.github/workflows/prevent-merge-if-release-open.yml

```diff
@@ -0,0 +1,29 @@
+name: Check for unmerged release PR
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: read
+
+jobs:
+  check_merge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for unmerged release
+        id: check_pr
+        uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const prs = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open'
+            })
+
+            const openRelease = prs.data.find(pr => pr.user.login === 'alphagov-pay-ci-concourse' && pr.state === 'open')
+
+            if (openRelease) {
+              core.setFailed('There is an unmerged release PR, please merge it before merging this PR.')
+            }
```

Comment:
Will this script work on the `ubuntu-latest` GHA runner? I think we need a `setup-node`